app/assets/json/translations/react/es.json

```diff
@@ -503,23 +505,24 @@
   "label.greatGrandparent": "Bisabuelo(a)",
   "label.hearingImpairments": "Para personas con impedimentos auditivos",
   "label.householdDetails": "Detalles del grupo familiar",
+  "label.householdMember": "Integrante del grupo familiar",
   "label.householdMemberCancel": "Cancelar la incorporación de esta persona",
   "label.householdMemberDelete": "Borrar a esta persona",
   "label.householdMemberName": "Nombre del integrante del grupo familiar",
   "label.householdMemberRelationship": "¿Qué parentesco tiene con usted?",
   "label.householdMemberSave": "Guardar al integrante del grupo familiar",
   "label.householdMemberUpdate": "Actualizar al integrante del grupo familiar",
-  "label.householdMember": "Integrante del grupo familiar",
   "label.householdMembers": "Integrantes del grupo familiar",
   "label.householdVeteran": "¿Es alguien más en esta solicitud un veterano?",
-  "label.howDidYouHearAboutThisSubtitle": "Díganos cómo se enteró de esta oportunidad de vivienda.",
   "label.howDidYouHearAboutThis": "¿Cómo se enteró sobre este aviso de vivienda?",
+  "label.howDidYouHearAboutThisSubtitle": "Díganos cómo se enteró de esta oportunidad de vivienda.",
   "label.howYouHeardAboutThisListing": "Cómo se enteró de este anuncio.",
   "label.inLaw": "Pariente político",
-  "label.incomePlaceholder": "Total de todas las fuentes de ingresos",
   "label.income": "Ingresos",
+  "label.incomePlaceholder": "Total de todas las fuentes de ingresos",
   "label.languageSpokenAtHome": "Idioma que se habla en casa",
   "label.lastName": "Apellido",
+  "label.less": " Menos",
```

Comment:
should this and line 538 have a leading space? I see it in the other languages too but just wanna make sure

Reply:
thanks for catching all of these! i wonder if there's a tool that could clean these up automagically

Reply:
i was able to find all the trailing whitespaces in vscode by regex searching `[ \t]",` but a nice automagic tool would be cool 🪄

Reply:
There's a bunch more preceding/trailing ws in strings not recently added. I decided to leave them as is for now and we should cleanup when we're working on them, or if someone has time to investigate each one. It looks like some of them may be intentional, in which case we may want to cleanup the markup and string properties when rewriting those components. Perhaps the ws would be better served with css or two strings can be collapsed into one with the space in between rather than trailing.